mappings/net/minecraft/util/JsonSerializing.mapping

```diff
@@ -1,29 +1,35 @@
 CLASS net/minecraft/class_5330 net/minecraft/util/JsonSerializing
-	METHOD method_29306 createTypeHandler (Lnet/minecraft/class_2378;Ljava/lang/String;Ljava/lang/String;Ljava/util/function/Function;)Lnet/minecraft/class_5330$class_5331;
+	METHOD method_29306 createSerializerBuilder (Lnet/minecraft/class_2378;Ljava/lang/String;Ljava/lang/String;Ljava/util/function/Function;)Lnet/minecraft/class_5330$class_5331;
 		ARG 0 registry
 		ARG 1 rootFieldName
 		ARG 2 idFieldName
 		ARG 3 typeIdentification
-	CLASS class_5331 TypeHandler
-		COMMENT A handler of JSON serializable types that can either obtain a type from
+	CLASS class_5331 SerializerBuilder
+		COMMENT A builder for serializing types to JSON that can either obtain a type from
 		COMMENT a registry to handle JSON conversion or handle with a custom logic bound
 		COMMENT to a type.
 		COMMENT
-		COMMENT <p>When the root element read is an object, the handler obtains the type
+		COMMENT <p>When the root element read is an object, the built serializer obtains the type
 		COMMENT from registry to handle reading; otherwise, it falls back to custom
 		COMMENT logic.
 		FIELD field_25192 registry Lnet/minecraft/class_2378;
 		FIELD field_25193 rootFieldName Ljava/lang/String;
 		FIELD field_25194 idFieldName Ljava/lang/String;
 		FIELD field_25195 typeIdentification Ljava/util/function/Function;
 		FIELD field_25196 customSerializer Lcom/mojang/datafixers/util/Pair;
+		FIELD field_28444 defaultType Lnet/minecraft/class_5336;
 		METHOD <init> (Lnet/minecraft/class_2378;Ljava/lang/String;Ljava/lang/String;Ljava/util/function/Function;)V
 			ARG 1 registry
 			ARG 2 rootFieldName
 			ARG 3 idFieldName
 			ARG 4 typeIdentification
-		METHOD method_29307 createGsonSerializer ()Ljava/lang/Object;
-	CLASS class_5332 CustomSerializer
+		METHOD method_29307 build ()Ljava/lang/Object;
+		METHOD method_32385 customSerializer (Lnet/minecraft/class_5336;Lnet/minecraft/class_5330$class_5332;)Lnet/minecraft/class_5330$class_5331;
+			ARG 1 type
+			ARG 2 serializer
+		METHOD method_33409 defaultType (Lnet/minecraft/class_5336;)Lnet/minecraft/class_5330$class_5331;
+			ARG 1 defaultType
+	CLASS class_5332 type
```

Comment:
```suggestion
	CLASS class_5332 Type
```

Reply:
Yeah no this should still have the old name, not sure what I did here...